code/modules/clothing/head/jobs.dm

```diff
@@ -146,19 +146,51 @@
 	dog_fashion = /datum/dog_fashion/head/warden
 
 /obj/item/clothing/head/warden/drill
-	name = "warden's drill hat"
-	desc = "A special armored campaign hat with the security insignia emblazoned on it. Uses reinforced fabric to offer sufficient protection. Has the letters 'FMJ' enscribed on its side."
+	name = "warden's campaign hat"
+	desc = "A special armored campaign hat with the security insignia emblazoned on it. Uses reinforced fabric to offer sufficient protection."
 	icon_state = "wardendrill"
 	item_state = "wardendrill"
 	dog_fashion = null
+	var/anger = 1
+
+/obj/item/clothing/head/warden/drill/screwdriver_act(mob/living/carbon/human/user, obj/item/I)
+	if(..())
+		return TRUE
+	switch(anger)
+		if(1)
+			to_chat(user, "<span class='notice'>You set the voice circuit to the middle position.</span>")
+			anger = 2
+		if(2)
+			to_chat(user, "<span class='notice'>You set the voice circuit to the last position.</span>")
+			anger = 3
+		if(3)
+			to_chat(user, "<span class='notice'>You set the voice circuit to the first position.</span>")
+			anger = 1
+		if(4)
+			to_chat(user, "<span class='danger'>You adjust voice circuit but nothing happens, probably because it's broken.</span>")
+	return TRUE
+
+/obj/item/clothing/head/warden/drill/wirecutter_act(mob/living/user, obj/item/I)
+	if(anger != 4)
+		to_chat(user, "<span class='danger'>You broke the voice circuit!</span>")
+		anger = 4
+	return TRUE
 
 /obj/item/clothing/head/warden/drill/equipped(mob/living/carbon/human/user, slot)
 	..()
-	if(slot == SLOT_HEAD)
+	if(anger == 2 && slot == SLOT_HEAD)
+		user.dna.add_mutation(SHOUTING)
+	if(anger == 3 && slot == SLOT_HEAD)
 		user.dna.add_mutation(YELLING)
+	if(anger == 4 && slot == SLOT_HEAD)
+		user.dna.add_mutation(CANADIAN)
+	else
+		return
 
 /obj/item/clothing/head/warden/drill/dropped(mob/living/carbon/human/user)

```

Comment:
should I be afraid equipped calls inher but dropped doesn't?

Reply:
denton told me to use dropped, so i trust his judgement for this

Reply:
just checked, it needs to call inheritance.

Reply:
since i'm using speechmod i'm not going to have to use dropped